app/javascript/packs/state-guidance.ts

```diff
@@ -10,11 +12,27 @@ export function showOrHidePuertoRicoExtras(forStateCode) {
   });
 }
 
+export function showOrHideJurisdictionExtras(forJurisdictionCode) {
+  document.querySelectorAll('.jurisdiction-extras').forEach((element) => {
+    switch (forJurisdictionCode) {
+      case 'TX':
+        element.innerHTML = t('in_person_proofing.form.state_id.state_id_number_texas_hint');
+        break;
+      case 'FL':
+        element.innerHTML = t('in_person_proofing.form.state_id.state_id_number_hint_html');
+        break;
```

Comment:
I'd consider breaking this into two helper functions? It's nice to be able to directly `return` from a `switch/case` statement so we don't have to worry about adding `break` and side effects and stuff, so like:

```js
  document.querySelectorAll('.jurisdiction-extras').forEach((element) => {
    element.innerHTML = jurisdictionExtraHelperText(forJurisdictionCode);
  });
  // ...


  function jurisdictionExtraHelperText(jurisdiction) {
    switch(jurisdiction) {
    case 'TX':
      return '...';
    default:
      return '...;
    }
  }
```

Reply:
that's definitely neater